src/Core_Command.php

```diff
@@ -1150,21 +1176,26 @@ function update_db( $_, $assoc_args ) {
 		if ( $network ) {
 			$iterator_args = array(
 				'table' => $wpdb->blogs,
-				'where' => array( 'spam' => 0, 'deleted' => 0, 'archived' => 0 ),
+				'where' => array(
```

Comment:
Similar to other places, you may want to consider declaring `$where` as a variable and passing it in.